### PR TITLE
Finalize the HISTORY in preparation for release.

### PR DESCRIPTION
**Pull request by jdmgoogle**
LAST ITEM

### CODE REVIEW

**Comment by jdmgoogle** [comment]
This addresses issue #266.

File: HISTORY.md [hunk 1]
```diff
@@ -60,6 +62,9 @@ This document includes release notes for the [VIP Spec][vip] for versions
   * Issue [#66](https://github.com/votinginfoproject/vip-specification/issues/66)
 
 ### Other Changes
+* The spec and all content in this repository is available under a [Creative
+  Commons With-Attribution](LICENSE.md) license.
```
Comment (cjerdonek) [inline HISTORY.md]: I would put the full license name here, because -- why not? :grinning:

**Comment by cjerdonek** [comment]
LGTM with my one suggestion.

**Comment by jdmgoogle** [comment]
License name is updated.

**Comment by jungshadow** [comment]
LGTM

**Comment by cjerdonek** [comment]
I'm expecting some good GIFs after this, guys..

**Comment by jungshadow** [comment]
I think this is gtg, @jdmgoogle, since @cjerdonek gave the LGTM with the change. So...
Bob Odenkirk throwing confetti

**Comment by jdmgoogle** [comment]
This closes out #266.

image

**Comment by jdmgoogle** [comment]
kraken

**Pull request merged**